packit_service/models.py

```diff
@@ -2624,6 +2624,8 @@ class KojiBuildTargetModel(GroupAndTargetModelConnector, Base):
     build_submitted_time = Column(DateTime, default=datetime.utcnow)
     build_start_time = Column(DateTime)
     build_finished_time = Column(DateTime)
+    # stdout for the Koji build submission command
```

Comment:
Just a nitpick, but I think this should be:

```suggestion
    # stdout from the Koji build submission command
```

or:

```suggestion
    # stdout of the Koji build submission command
```